packages/gateway/src/features/user/root-resolvers.ts

```diff
@@ -466,6 +466,38 @@ export const resolvers: GQLResolver = {
       }
 
       return true
+    },
+    async resetPasswordSMS(_, { userId, applicationName }, authHeader) {
+      if (!hasScope(authHeader, 'sysadmin')) {
+        return await Promise.reject(
+          new Error(
+            'Reset password can only be sent by a user with sys admin scope'
+          )
+        )
+      }
+
+      const res = await fetch(`${USER_MANAGEMENT_URL}resetPasswordSMS`, {
+        method: 'POST',
+        body: JSON.stringify({
+          userId,
+          applicationName
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+          ...authHeader
+        }
+      })
+
+      if (res.status !== 200) {
+        return await Promise.reject(
+          new Error(
+            `Something went wrong on user-mgnt service. Couldn't reset password and send sms to ${userId}`
+          )
+        )
+      }
+
+      const response = await res.json()
+      return response.userName
```

Comment:
I thought we might need the name of the user to be returned form the mutation to be able to show it on the success notification. But it seems that it's not needed so let's just return a boolean value from here. We can also amend the handler in user-mgnt to not return the username as well

Reply:
Yeah. I was also thinking the same.